### ISSUE TITLE
owl component without returning HTML

### ISSUE THREAD

**Issue by msh-odoo** [odoo/owl#710]
I have case where I don't want to render anything if value is not there in component, consider following example.

```
class myComponent extends owl.Component {
    constructor(parent, value) {
        this.value = value;
    }
}
myComponent.template = xml`<div t-if="value" class="someClass">
                                Hello World!
                            </div>`;
```

Is it possible as it gives warning component doesn't return anything, is there way to achieve it or workaround?

@ged-odoo @aab-odoo

**Comment by jat-odoo** [comment]
@msh-odoo Maybe this will help.

myComponent.template = xml`<t><div t-if="value" class="someClass">
                                Hello World!
                            </div></t>`;

**Comment by ged-odoo** [comment]
@jat-odoo  this will probably not work. A component needs a root node.

For Owl 2.0, we will support multiroots, and possibly zero roots, but this is really much more difficult, because we need to be able to reconcile the DOM after each update.  Usually, we can move the `t-if` in the parent component, and then we don't have a problem.

**Comment by jat-odoo** [comment]
@ged-odoo, I did't read the OWL documentation carefully yet.
But i assume that `<t>` tag will work as `<React.Fragment>`. BTW similar like fragment maybe available in OWL 2.0 that's nice.

**Comment by ged-odoo** [comment]
fixed in owl 2
